Gemfile

```diff
@@ -13,7 +13,8 @@ gem 'uglifier', '2.7.1'
 gem 'coffee-rails', '4.0.1'
 # See https://github.com/sstephenson/execjs#readme for more supported runtimes
 # gem 'therubyracer',  platforms: :ruby
-
+# Gem for localization
+gem 'rails-i18n'
```

Comment:
Please specify gem version. Use 'bundle' command to check the current, and use it in the file.